ci/prow-entrypoint.sh

```diff
@@ -233,53 +227,45 @@ main () {
             ;;
         "rhcos-cosa-prow-pr-ci")
             setup_user
-            cosa_init
+            cosa_init "rhel-coreos-8"
```

Comment:
Bikeshed: WDYT about `rhcos8` and `rhcos9` instead? Seems closer to what we're used to type and matches the other `scos` better too.

Reply:
See https://github.com/coreos/coreos-assembler/pull/2934#discussion_r952978285

Reply:
I agree typing `rhcos` is more convenient in quick emails and interactive chat.  But for anything a bit more formal I'd like to move to being more explicit because the world can only have so many acronyms.  "rhel-coreos" is not *that* much longer and is much, much more self-describing, even though it still contains one acronym.

Reply:
We've started using "rhel-coreos-8" & "rhel-coreos-9" for the machine-os-content & container names somewhere I don't remember exactly thus the change here. CC @cgwalters

Reply:
https://github.com/openshift/release/blob/master/ci-operator/config/openshift/os/openshift-os-master__periodic.yaml#L101

Reply:
I think my core argument here is: With the introduction of RHEL9, the abbreviation "rhcos" has become ambiguous outside of the most generic contexts.  Specifically, mirror.openshift.com for example can't just use `rhcos` - it has to become either `rhcos8` and `rhcos9` as you suggest, but I think while we are renaming things, let's use `rhel-coreos-8` and `rhel-coreos-9` there and in our disk images in general.

Reply:
No strong argument against `rhel-coreos-N`, but note that RHCOS as an acronym is pretty established at this point, including in official OpenShift docs. We do need to start adding the version string, but I'm not sure it follows that we need to also change the prefix.